code/style.css

```diff
@@ -1,7 +1,487 @@
 
-.active + .answer {
-  /* some styling here */
+*,
+*:before,
+*:after {
+  box-sizing: inherit;
+}
+
+body {
+  box-sizing: border-box;
+  min-width: 350px;
+  font-family: "Roboto", sans-serif;
+  margin: 0;
+  padding: 0;
+  background-color: rgb(156, 212, 214);
+}
+
+/* -------HEADER CONTENT-------- */
+
+.header {
+  height: 100vh;
+  width: 100%;
+}
+
+.header__title {
+  position: absolute;
+  display: flex;
+  justify-content: flex-end;
+  flex-direction: column;
+  align-items: center;
+  height: 100vh;
+  width: 100%;
+  background-color: rgb(0, 0, 0, 0.25);
+  min-width: 350px;
+}
+
+h1 {
+  text-align: center;
+  font-family: "Cinzel Decorative", cursive;
+  font-weight: 100;
+  color: rgb(255, 255, 255);
+  margin-bottom: 3%;
+  font-size: 6vw;
+}
+
+.header__title__arrow {
+  color: white;
+  margin-bottom: 50px;
+  font-size: 2vw;
+  z-index: 10;
+}
+
+.header__video {
+  height: 100vh;
+  width: 100%;
+  z-index: -1;
+  object-fit: cover;
+}
+
+.header__video--mobile {
+  display: flex;
+}
+
+.header__video--ipad {
+  display: none;
+}
+
+.header__video--desktop {
+  display: none;
+}
+
+.navbar {
+  width: 100%;
+  height: 80px;
+  background-color: rgb(0, 0, 0);
+  color: white;
+  display: grid;
+  grid-template-columns: 1fr auto auto auto;
+  align-items: center;
+  padding: 0 3%;
+  column-gap: 10px;
+  position: sticky;
+  top: 0;
+  z-index: 1;
+}
+
+.navbar__logotype {
+  height: 30px;
+  width: 200px;
+}
+
+.navbar__item {
+  text-align: center;
+  display: grid;
+  grid-template-rows: auto auto;
+  row-gap: 10px;
+  text-decoration: none;
+  color: white;
+  font-size: 14px;
+}
+
+.navbar__item:hover {
+  color: rgb(255, 147, 6);
+  transition: color 1s ease-in;
+  border-bottom: 1px solid rgb(255, 147, 6);
+}
+
+.navbar__item a:active {
+  transition: all 1s ease-in;
+  color: rgb(255, 147, 6);
+  border-bottom: 1px solid rgb(255, 147, 6);
+}
+
+.form-accordion-contact-container {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+}
+
+.hidden-anchor {
+  display: block;
+  position: relative;
+  top: -80px;
+  visibility: hidden;
+}
+
+/*------------- FORM CONTAINER ---------------- */
+
+.form {
+  padding: 30px 0;
+  display: grid;
+  max-width: 1200px;
+  margin: auto;
+}
+
+.form__content {
+  padding: 0 10%;
+}
+
+h2 {
+  margin-bottom: 20px;
+  padding: 0;
+  width: 100%;
+}
+
+.form__content__layout {
+  display: grid;
+  row-gap: 10px;
+}
+
+.form__content__layout__row {
+  font-size: 16px;
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+}
+
+.form__content__layout__row__input-text {
+  padding: 5px 5px 5px;
+  -webkit-appearance: none;
+  background-color: rgb(255, 254, 253, 0);
+  border: 0;
+  border-bottom: 1px solid gray;
+  font-size: 16px;
+  background-color: white;
+}
+
+.form__content__layout__row__input-text:focus,
+.form__content__layout__row__room-selector:focus,
+.form__content__layout__row__date-picker:focus {
```

Comment:
I got a little bit lost with the classes in the form. With this viewing option that you cannot put them side by side, it was a little bit confusing to have big classes names that were so similar to keep going back and forth.